### DISCUSSION TITLE
Found no NVIDIA driver on your system, but I have a dedicated nvidia GPU (in unraid)

### DISCUSSION THREAD

**Discussion by cdponcedeleon** [AbdBarho/stable-diffusion-webui-docker#501] [Q&A]
Hey community, good day to everyone!

I am having some issues running this in docker (running in unraid). whenever I start the container I get the following warning 

Warning: caught exception 'Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx', memory monitor disabled

However, my server has an nvidia card with drivers as well, I had a different deployment of Stable diffusion and it worked with the GPU card with no issues. Is there anything in particular that I need to do for the container to see the card?

Thanks!

**Comment by AbdBarho** [comment]
What is your OS? GPU? do you have the `nvidia-container-cli` installed?
have you checkout the FAQ? https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ

**Reply by cdponcedeleon** [reply]
OS is Unraid, I do have the nvidia-container-cli installed, is using the following drivers

Nvidia Driver Version: | 530.41.03
Open Source Kernel Module: | No
Installed GPU(s): | 0: NVIDIA GeForce RTX 3060 05:00.0 GPU-274c4c48-212d-ff9b-3c06-a63da44255c5

My only thought is that perhaps I need to manually assign the driver to the container? Is there a variable I can modify for it?

Thanks for answering! Any other suggestion is greatly appreciated.

**Reply by AbdBarho** [reply]
what is the output of this command?
```
docker run --rm --runtime=nvidia --gpus all nvidia/cuda:11.6.2-base-ubuntu20.04 nvidia-smi
```

**Reply by cdponcedeleon** [reply]
This is the output of the command
```
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 530.41.03              Driver Version: 530.41.03    CUDA Version: 12.1     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                  Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf            Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 3060         Off| 00000000:05:00.0 Off |                  N/A |
| 30%   28C    P0               30W / 170W|      0MiB / 12288MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
```
Thanks for all the help!
